requirements.in

```diff
@@ -9,7 +9,8 @@ dj-static==0.0.6
 django-redis==4.7.0
 django-server-status==0.3.2
 django-webpack-loader==0.4.1
-djangorestframework==3.5.2
+djangorestframework==3.9.1
+django-debug-toolbar==1.11
```

Comment:
Why is this added here and also below in `test_requirements.txt`?

Reply:
@noisecapella newer version of `django-debug-toolbar` was expecting `MIDDLEWARE` in settings instead of `MIDDLEWARE_CLASSES`. For compatibility I have to pin its version here and in test requirements.

Reply:
Why does it need to be here and also in `test_requirements.txt`? Why not just pin it in `test_requirements.txt`?

Reply:
@noisecapella My bad, I thought I have to update here too to run project smooth. I have removed this from `requirements.txt`.